### ISSUE TITLE
Not able to install Google Chrome in Mac Executor

### ISSUE THREAD

**Issue by abhinaba-ghosh** [CircleCI-Public/browser-tools-orb#44]
### Orb version
circleci/browser-tools@1.2.5

### What happened
The Job - 
```
steps:
  - checkout
  - macos/add-uitest-permissions
  - run: 
       command: HOMEBREW_NO_AUTO_UPDATE=1 brew install wget
        name: 'Install wget'
  - browser-tools/install-chrome:
            chrome-version: "latest"
```

The installation of chrome shows below error - 
<img width="1158" alt="Screenshot 2022-04-01 at 11 20 09 PM" src="https://user-images.githubusercontent.com/23141842/161316958-17d130dc-78e8-4307-8f0b-e7cb7f195833.png">

### Expected behaviour

Chrome should be installed.

**Comment by katerberg** [comment]
This also occurs for me on Linux.

version: 2.1
orbs:
  aws-ecr: circleci/aws-ecr@6.7.1
  aws-s3: circleci/aws-s3@2.0.0
  browser-tools: circleci/browser-tools@1.2.5
      - image: cimg/ruby:2.7.5-browsers
    steps:
      - browser-tools/install-chrome

**Comment by abhinaba-ghosh** [comment]
Any update/guidance for this?

**Comment by KyleTryon** [comment]
Hello all,

I have a PR open here that aims to resolve this issue. If anyone is interested, you can help us out by testing out this version of the orb before it goes live.

https://github.com/CircleCI-Public/browser-tools-orb/pull/48#issuecomment-1099241298

**Comment by KyleTryon** [comment]
Hello,

We have just released version 1.3 of the browser-tools orb with this issue in mind. Please try this version and let us know if you experience any issues.

https://github.com/CircleCI-Public/browser-tools-orb/releases/tag/v1.3.0
https://circleci.com/developer/orbs/orb/circleci/browser-tools?version=1.3.0
